github-runner-provisioner/config.go

```diff
@@ -1,7 +1,8 @@
 package main
 
 type Config struct {
-	GithubToken string `required:"true" envconfig:"GITHUB_TOKEN"`
+	GithubToken  string `required:"true" envconfig:"GITHUB_TOKEN"`
+	WebhookToken string `required:"true" envconfig:"WEBHOOK_TOKEN"`
```

Comment:
We should document what are these variables and the provisioner architecture.
I'll add a note to the Cycle page

Reply:
Updated the README to clarify this.